changelog/README.md

```diff
@@ -0,0 +1,34 @@
+How to add a new changelog entry to the pending changelog?
+==========================================================
+
+This will get copied to dlang.org and cleared when master gets
+merged into stable prior to a new release.
+
+```
```

Comment:
> I find this very confusing

Please see https://github.com/dlang/phobos/pull/4228 for the motivation and examples.

> Is what follows the content of the file?

Yes.

> If so, what's the name of the file? 

There's no fixed scheme except that the extension should be `.dd` (Ddoc extension).

> Maybe add more documentation around here.

Ok - Will do so later.